### ISSUE TITLE
GUI options for changing ANSI colours

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#193]
```
It would be great if there were options to change the colors that are actually 
displayed when showing ANSI color.  This is the same feature that is in PuTTY 
under Window -> Colours.

An example use for this is that by default vim and other programs render bash 
comments in blue, and the default blue (0,0,127) is almost impossible to read 
on a black background.  I would like to be able to change the color of this 
blue or make it brighter by adjusting the RGB values that are displayed.
```

Original issue reported on code.google.com by `brian.ma...@gmail.com` on 30 Jun 2010 at 4:02

**Comment by mintty** [comment]
The new option `-C` (capital C) or `--loadconfig` loads an additional configuration file which will not be used for saving configuration changes.
This is particularly intended to mix-in colour schemes (or other style schemes).
See https://github.com/mintty/mintty/wiki/Tips for some details.

**Comment by mintty** [comment]
Introduced ThemeFile setting in 2.2.4, including GUI option.

**Comment by mintty** [comment]
Mintty now supports loading and storing colour schemes from the Color Scheme Configurator in its Options menu.

**Comment by mintty** [comment]
Amended, polished and finished GUI configuration of colour schemes. Released with 2.4.0.

**Comment by mintty** [comment]
Released 2.7.6.
